master/internal/core_experiment.go

```diff
@@ -341,7 +341,7 @@ func (m *Master) patchExperiment(c echo.Context) (interface{}, error) {
 
 // CreateExperimentParams defines a request to create an experiment.
 type CreateExperimentParams struct {
-	ConfigBytes   string          `json:"experiment_config"`
+	ConfigBytes   *string         `json:"experiment_config"`
```

Comment:
why should we make this a pointer just to deref it sort of unsafely all over?

Reply:
Hmm other than trying to save a few copies I was thinking this could allow us to potentially save an experiment with no user config 🤔 Maybe something we could use for forks?  But I also forgot to check for nil pointers

Reply:
this doesn't seem to be on a perf critical code path so it might be better to stick with the simpler solution and copy?

Reply:
go strings are copy on modification afaik so sholdn't be any copies since this value is read only.

Reply:
even so yeah, i think it's fine